components/Textarea/Textarea.tsx

```diff
@@ -215,20 +220,35 @@ const TextareaWrapper = styled('div', {
   },
 });
 
-export interface TextareaVariants extends VariantProps<typeof StyledTextarea> { }
+const AdornmentWrapperEnd = styled('div', {
+  position: 'absolute',
+  bottom: '$2',
+  right: '$3',
+  minWidth: '$5',
+  minHeight: '$5',
+  zIndex: 1,
```

Comment:
> ❓  Question

Will this work in modals?

Reply:
@paulocfjunior elements with a `z-index` stack on top of their parent's index.

Therefore, if we have a `Textarea` child of a `Dialog`, it will have the Dialog's z-index + 1.

The only cases where it might be an issue are therefore if we don't have a `zIndex` high enough for elements which should be "on top" of others.

:information_source: Dialogs have the highest z-index, for now, as they're children of a portal.

![image](https://user-images.githubusercontent.com/3367393/155031961-a8f90669-9a4d-4203-a124-848ad4899562.png)